setup/config/webpack.config.js

```diff
@@ -5,12 +5,13 @@ const config = {
   mode: 'development',
   resolve: {
     alias: {
+      src: resolve('./src'),
```

Comment:
@soroushm I believe this should be `'@src'`

Reply:
we can have the src because its not a modules its a root project